api/migrations/0001_initial.py

```diff
@@ -7,18 +7,25 @@ class Migration(migrations.Migration):
 
     initial = True
 
```

Comment:
It's just reformatting it. If we don't push it will be reformatted and probably added by accident later. Or I can figure out a way to exclude the migration files from being formatted?

Reply:
Right. Formatting should be fine. I just saw a migration file and panicked